### DISCUSSION TITLE
Apache2, rewrite, and SSL

### DISCUSSION THREAD

**Discussion by gvjensen** [magicbug/Cloudlog#1075] [Q&A]
Guys - 

I have been having a devil of time trying to get Apache to serve Cloudlog (with SSL support / rewrite).

I get errors saying that I have mixed content.  Moving forward, I lose SSL and the base url is not rewritten.

**_(after I accept a security exception):_**

![image](https://user-images.githubusercontent.com/13772978/123560302-3d44a400-d756-11eb-89cd-8a0457069424.png)

**_After I login:_** 

![image](https://user-images.githubusercontent.com/13772978/123560419-d378ca00-d756-11eb-85ce-8a1e836a4f4a.png)

I know this may be something super simple, but I have been wracking my brain with .htaccess and Apache2 virtual server configuration changes.

My server supports a number of virtual servers, with SSL (Let's Encrypt).

Ideas?

73's - Glenn

**Comment by enochianborg** [comment]
In your Installation, go to the Cloudlog/application/config then open config.php with your editor.  In the file click search for what looks like this $config['base_url']	= 'http://n7mwh.bounceme.net/cloudlog/'; and add an s after http and your ssl enabled server will serve it as https with the certificate you have installed in your server certificate if you already have that set up properly.  It takes some knowledge to get to that point and or your own self signed certificate which will still show NOT secured while on the site but it will be https and basically faking the CA certificate until your cert is validated and installed to your server.
